spec/integration/cypress/plugins/index.js

```diff
@@ -16,7 +16,7 @@ const cucumber = require("cypress-cucumber-preprocessor").default
 /**
  * @type {Cypress.PluginConfig}
  */
-module.exports = (on, config) => {
+module.exports = (on) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
```

Comment:
```suggestion
```

Reply:
I removed this because with the updated linter it threw an `unused variable` error

Reply:
That ok, just remove the comment too 😄

Reply:
Ah I misunderstood :) good catch